GliaWidgets/Public/Glia/Glia.swift

```diff
@@ -37,6 +37,21 @@ extension EngagementKind {
             return false
         }
     }
+
+    var mediaType: CoreSdkClient.MediaType {
```

Comment:
It seems like the logic for enqueueing is no longer dependant on `MediaType`, but on `EngagementKind`?
Maybe this mapping can encapsulated in `CoreSdkClient.MediaType` extension, and used (`MediaType(self)`) here? Like so:
Something like:
```swift
extension CoreSdkClient.MediaType {
    init(_ kind: EngagementKind) {
        switch kind {
            ....
        }
    }
}
```
Also it would make sense to add unit test for it.

Reply:
CoreSDKClient.MediaType extended [18d25dc](https://github.com/salemove/ios-sdk-widgets/pull/1179/commits/18d25dcc689f4b46be4c94c785f8db76c0a1ce7b)